src/content/en/updates/2018/12/badging-api.md

```diff
@@ -61,54 +68,68 @@ Examples of sites that may use this API include:
 | **3. Gather feedback & iterate on design** | [**In progress**](#feedback) |
-| 4. Origin trial                            | Not started                  |
+| **4. Origin trial**                        | [**In progress**](#ot)       |
 | 5. Launch                                  | Not started                  |
 
+
 ## How to use the badging API {: #use }
 
-Dogfood: We are still iterating on the design of the Badging API, and it’s
-**not** available in the browser yet. The sample code you see is based on
-the current design, and will likely change between now and the time it lands
-in the browser.
+Starting in Chrome 73, the Badging API is available as an origin trial
+for (?PLATFORMS?).
+[Origin trials][ot-what-is] allow you to try out new features and give
+feedback on usability, practicality, and effectiveness to us, and the web
+standards community. For more information, see the
+[Origin Trials Guide for Web Developers][ot-dev-guide].
+
+### Register for the origin trial {: #ot }
+
+1. [Request a token][ot-request] for your origin.
+2. Add the token to your pages, there are two ways to provide this token on
+   any pages in your origin:
+     - Add an `origin-trial` `<meta>` tag to the head of any page. For example,
+       this may look something like:
+       `<meta http-equiv="origin-trial" content="TOKEN_GOES_HERE">`
+     - If you can configure your server, you can also provide the token on pages
+       using an `Origin-Trial` HTTP header. The resulting response header should
+       look something like: `Origin-Trial: TOKEN_GOES_HERE`
+
+
+### Using the badging API during the origin trial
+
+Dogfood: During the origin trial, the API will be available via
+`window.ExperimentalBadging`. The below code is based on the current design,
+and will change before it lands in the browser as a standardized API.
 
 To use the badging API, your web app needs to meet
 [Chrome’s installability criteria](/web/fundamentals/app-install-banners/#criteria),
 and a user must add it to their home screen.
 
-The `Badge` interface is a member object on `Window` and `Worker`. It contains
+The `ExperimentalBadge` interface is a member object on `window`. It contains
 two methods:
 
-* `set([number])`: Sets the app's badge. If a value is provided, set the
-  badge to the provided value.
+
+* `set([number])`: Sets the app's badge. If a value is provided, set the badge
+  to the provided value otherwise, display a plain white dot (or other flag as
+  appropriate to the platform).
 * `clear()`: Removes app's badge.
 
 
 ```js
 // In a web page
 const unreadCount = 24;
-window.Badge.set(unreadCount);
-
-// In a service worker
-self.addEventListener('sync', () => {
-  self.Badge.set(getUnreadCount());
-});
+window.ExperimentalBadge.set(unreadCount);
 ```
 
+`ExperimentalBadge.set()` and `ExperimentalBadge.clear()` can be called from
+a foreground page, or potentially in the future, a service worker. In either
```

Comment:
I tweaked the wording here to indicate the it **may** be available in service workers in the future.